tests/resolvers/Post/creator.spec.ts

```
@@ -37,6 +38,10 @@ describe("resolvers -> Post -> creatorId", () => {
       _id: testPost!.creatorId,
     }).lean();
 
+    if (creatorIdObject && creatorIdObject.email) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)